target/linux/ramips/image/mt7621.mk

```diff
@@ -275,6 +275,26 @@ define Device/dlink_dir-1960-a1
 endef
 TARGET_DEVICES += dlink_dir-1960-a1
 
+define Device/dlink_dir-853-a3
+  $(Device/dsa-migration)
```

Comment:
It appears you can just use the dlink_dir-xx60-a1 definition here as well, although the name doesn't match.

Reply:
> So, this device requires soldering to be flashed?

Yes it does. The emergency room does not work using the reset button (but works from serial).

Reply:
> It appears you can just use the dlink_dir-xx60-a1 definition here as well, although the name doesn't match.

Yes and I did use that but since they share a common dtsi as well I thought it would be better to treat it as a separate device.

Reply:
I went ahead and made the changes.